web/ui/dashboard/src/app/private/pages/project/events/event-deliveries/event-deliveries.component.html

```diff
@@ -217,6 +217,7 @@ <h4 class="mb-10px font-semibold text-12">Deliveries missing?</h4>
 									size="xs"
 									texture="light"
 									[disabled]="event.status !== 'Success' && event.status !== 'Failure' && event.status !== 'Discarded'"
+									[title]="event.status !== 'Failure' && event.status !== 'Success' && event.status !== 'Discarded' ? 'You can not retry an event when it is scheduled or still processing.' : ''"
```

Comment:
This should be replay, not retry

Reply:
This is for an event delivery, aren't event deliveries retried while events are replayed